### PR TITLE
prepare windows eventlog

### PR DESCRIPTION
**Pull request by daiksy**

### CODE REVIEW

File: wix/wrapper/wrapper_windows.go [hunk 1]
```diff
@@ -45,6 +45,10 @@ func main() {
 		return
 	}
 
+	err := eventlog.InstallAsEventCreate(name, eventlog.Error|eventlog.Warning|eventlog.Info)
+	if err != nil {
+		log.Fatal("SetupEventLogSource() failed: %s", err)
```
Comment (astj) [inline wix/wrapper/wrapper_windows.go]:
```
wix\wrapper\wrapper_windows.go:50: possible formatting directive in Fatal call
```

You should use `Fatalf` instead. ref: https://github.com/stretchr/testify/issues/339

**Comment by astj** [comment]
eventlog.InstallAsEventCreate seems to be called on install, but isn't it enough?
https://github.com/mackerelio/mackerel-agent/blob/master/wix/wrapper/wrapper_windows.go#L38 =>  https://github.com/mackerelio/mackerel-agent/blob/master/wix/wrapper/install.go#L62

**Comment by astj** [comment]
:point_up: https://github.com/mackerelio/mackerel-agent/pull/319#issuecomment-274708472 wrong. :sob:`wrapper.exe install` is for debugging, not for production install

**Comment by astj** [comment]
LGTM!

**Comment by mattn** [comment]
LGTM :+1:

**Comment by daiksy** [comment]
thanks!!

**Pull request merged**